packages/frontend/pages/device/[deviceKey].vue

```diff
@@ -33,15 +33,14 @@ const deviceKey = route.params.deviceKey;
                 <!-- Didn't use button componenet here, couldn't get the link to work with it -->
 
                 <button class="btn mt-1 bg-iris text-white me-4 px-4"><a :href="`/provenance/${route.params.deviceKey}`" style="color: white; text-decoration: none">View Provenance Records</a></button>
-                <button class="btn mt-1 bg-sky px-5" @click="getQRCode">Download QR Code</button>
+                <!-- <button class="btn mt-1 bg-sky px-5" style="font-size:18px;" @click="getQRCode">Download QR Code</button> -->
+                <p style="padding-top: 10px;">Click the QR code to download image.</p>
```

Comment:
Have you tested this on a mobile screen such as with Chrome dev tools? Adding fixed pixel sizes will look different on different screens.